interface/forms/misc_billing_options/new.php

```diff
@@ -55,137 +56,223 @@
 
 ?>
 <html>
-<head>
-<?php html_header_show(); ?>
+  <head>
+    <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+    <link rel="stylesheet" href="<?php echo $GLOBALS['webroot'] ?>/library/css/jquery.datetimepicker.css">
+    <!-- Get Bootstrap -->
+    <?php call_required_libraries(['bootstrap']); ?>
+    <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>/jquery-min-1-9-1/index.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>/bootstrap-3-3-4/dist/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery.datetimepicker.full.min.js"></script>
```

Comment:
Jake you will need to use the "call_required_libraries" for these items also.
This example is from patient_tracker
'''<?php
    //  Include Bootstrap and DateTimePicker
  call_required_libraries(array("jquery-min-3-1-1","bootstrap","datepicker"));
?>
'''

Reply:
Use the developer tools in chrome and you will see a console error

Reply:
Did I get it @teryhill ? My console looks fine now.